test/lib/cli/update-notifier.js

```diff
@@ -2,21 +2,60 @@ const t = require('tap')
 const { basename } = require('node:path')
 const tmock = require('../../fixtures/tmock')
 const mockNpm = require('../../fixtures/mock-npm')
+const MockRegistry = require('@npmcli/mock-registry')
+const pacote = require('pacote')
+const mockGlobals = require('@npmcli/mock-globals')
 
 const CURRENT_VERSION = '123.420.69'
 const CURRENT_MAJOR = '122.420.69'
 const CURRENT_MINOR = '123.419.69'
 const CURRENT_PATCH = '123.420.68'
 const NEXT_VERSION = '123.421.70'
+const NEXT_VERSION_ENGINE_COMPATIBLE = '123.421.60'
+const NEXT_VERSION_ENGINE_COMPATIBLE_MINOR = `123.420.70`
+const NEXT_VERSION_ENGINE_COMPATIBLE_PATCH = `123.421.58`
 const NEXT_MINOR = '123.420.70'
 const NEXT_PATCH = '123.421.69'
 const CURRENT_BETA = '124.0.0-beta.99999'
 const HAVE_BETA = '124.0.0-beta.0'
 
+const pacumentResponse = {
```

Comment:
packument not pacument (I didn't do a code suggestion cause it will need to be changed in multiple places)